Building error: m.Namespace.Key undefined (type plugin.Namespace has no field or method Key)

<!--
Thanks for filing an issue! Before hitting the button, give this a read.

If you are reporting a new issue, make sure that we do not have any duplicates already open. You can ensure this by searching the issue list for this repository. If there is a duplicate, please close your issue and add a comment to the existing issue instead.

If you suspect your issue is a bug, please add as much context as you can using the template below. If it is not a bug, remove any sections you don't need and explain how we can help.

As a basic rule, if you cannot provide enough information to continue addressing the issue within 7 days, a maintainer will close it. We will, however, reopen it if you later provide the information. Thanks again.
-->

**Snap version** (use `snapctl -v`): 
```
snapctl version master-eb4b5d8
```

**Environment**:
- **OS** (e.g. from /etc/os-release):
```
 /etc/os-release
NAME="Ubuntu"
VERSION="16.04.1 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.1 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
UBUNTU_CODENAME=xenial
```
- **Kernel** (e.g. `uname -a`): 
```
Linux kzab-Z97X-UD7-TH 4.4.0-42-generic #62-Ubuntu SMP Fri Oct 7 23:11:45 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```


**What happened**:
Executing `make` I'm receiving the following error:
```
2016-11-08 14:06:07 UTC [     info] building plugin: snap-plugin-publisher-graphite
# github.com/intelsdi-x/snap-plugin-publisher-graphite/graphite
graphite/graphite.go:73: m.Namespace.Key undefined (type plugin.Namespace has no field or method Key)
Makefile:37: recipe for target 'all' failed
make[1]: *** [all] Error 1
make[1]: Leaving directory '/home/kzab/work/src/github.com/intelsdi-x/snap-plugin-publisher-graphite'
Makefile:20: recipe for target 'default' failed
make: *** [default] Error 2
```
The plugin is not compatible with the latest version of snap-plugin-lib-go, `Key()` method was removed.

**Steps to reproduce it** 

1. Execute `make`

--- a/glide.lock
+++ b/glide.lock
@@ -1,0 +1,64 @@
+hash: bdb35fb3e772df1406ebbc495d9d9b77b02d2f729b48527926c857070d431caf
+updated: 2016-11-09T13:35:17.338229481-08:00
+imports:
+- name: github.com/golang/protobuf
+  version: 888eb0692c857ec880338addf316bd662d5e630e
+  subpackages:
+  - proto
+- name: github.com/intelsdi-x/snap
+  version: 89ba8ab9e12640de3c513f3e821a42771513b891
+  subpackages:
+  - control/plugin
+  - control/plugin/cpolicy
+  - core/ctypes
+- name: github.com/intelsdi-x/snap-plugin-lib-go
+  version: d7be15167858210a70ff958a5844acd93f68d989
+  subpackages:
+  - v1/plugin
+  - v1/plugin/rpc
+- name: github.com/marpaia/graphite-golang
+  version: c98f562aa632f89588e321a4f6013c3ae57aa48c
+- name: github.com/Sirupsen/logrus
+  version: d26492970760ca5d33129d2d799e34be5c4782eb
+- name: golang.org/x/net
+  version: 04557861f124410b768b1ba5bb3a91b705afbfc6
+  subpackages:
+  - context
+  - http2
+  - http2/hpack
+  - internal/timeseries
+  - lex/httplex
+  - trace
+- name: golang.org/x/sys
+  version: 8f0908ab3b2457e2e15403d3697c9ef5cb4b57a9
+  subpackages:
+  - unix
+- name: google.golang.org/grpc
+  version: 0032a855ba5c8a3c8e0d71c2deef354b70af1584
+  subpackages:
+  - codes
+  - credentials
+  - grpclog
+  - internal
+  - metadata
+  - naming
+  - peer
+  - transport
+testImports:
+- name: github.com/gopherjs/gopherjs
+  version: 4b53e1bddba0e2f734514aeb6c02db652f4c6fe8
+  subpackages:
+  - js
+- name: github.com/jtolds/gls
+  version: 8ddce2a84170772b95dd5d576c48d517b22cac63
+- name: github.com/smartystreets/assertions
+  version: 443d812296a84445c202c085f19e18fc238f8250
+  subpackages:
+  - internal/go-render/render
+  - internal/oglematchers
+- name: github.com/smartystreets/goconvey
+  version: d4c757aa9afd1e2fc1832aaab209b5794eb336e1
+  subpackages:
+  - convey
+  - convey/gotest
+  - convey/reporting

--- a/glide.yaml
+++ b/glide.yaml
@@ -2,13 +2,10 @@ package: github.com/intelsdi-x/snap-plugin-publisher-graphite
 import:
 - package: github.com/Sirupsen/logrus
   version: ^0.10.0
-- package: github.com/intelsdi-x/snap
-  version: ^0.16.1-beta
-  subpackages:
-  - control/plugin
-  - control/plugin/cpolicy
-  - core/ctypes
 - package: github.com/marpaia/graphite-golang
+- package: github.com/intelsdi-x/snap-plugin-lib-go
+  subpackages:
+  - v1/plugin
 testImport:
 - package: github.com/smartystreets/goconvey
   version: ^1.6.2

--- a/graphite/graphite.go
+++ b/graphite/graphite.go
@@ -70,7 +70,7 @@ func (f *GraphitePublisher) Publish(metrics []plugin.Metric, cfg plugin.Config) 
 
 	giteMetrics := make([]graphite.Metric, len(metrics))
 	for i, m := range metrics {
-		key := m.Namespace.Key()
+		key := strings.Join(m.Namespace.Strings(), ".")
 		for _, tag := range tagsForPrefix {
 			nextTag, ok := m.Tags[tag]
 			if ok {

